src/lib/components/wizard-navigation-bar.component.vertical.less

```diff
@@ -31,368 +31,191 @@
-          .state-circle-with-background-and-content-hover(@wz-color-default);
-        }
-
-        div {
-          min-height: @big-dot-height;
-        }
-      }
-
-      li.current .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-current);
-      }
-      li.done .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-done);
-      }
-      li.optional .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-optional);
-      }
-      li.editing .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-editing);
-      }
-
-      li.current a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-current);
-      }
-      li.done a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-done);
-      }
-      li.optional a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-optional);
-      }
-      li.editing a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-editing);
-      }
 
-      // default steps shouldn't change when hovered, because they aren't clickable
-      li.default a:hover .step-indicator {
-        .state-circle-with-background-and-content-nohover(@wz-color-current);
-      }
+    a:hover .step-indicator {
+      .state-circle-hover(@width, @height, @border);
+      .state-hover(@wz-color-default);
+    }
+    &.current a:hover .step-indicator {
+      .state-hover(@wz-color-current);
+    }
+    &.done a:hover .step-indicator {
+      .state-hover(@wz-color-done);
+    }
+    &.optional a:hover .step-indicator {
+      .state-hover(@wz-color-optional);
+    }
+    &.editing a:hover .step-indicator {
+      .state-hover(@wz-color-editing);
     }
   }
 
-  &.large-empty-symbols {
-    ul.steps-indicator {
-      padding: (@distance-between-steps / 2) (@distance-between-steps / 2) (@distance-between-steps / 2) (@distance-between-steps / 2 + @dot-baseline-distance + @big-dot-height);
-
-      li {
-        &:not(:last-child):before {
-          .line(@big-dot-width, @big-dot-height, @wz-color-default);
-        }
+  // default steps shouldn't change when hovered, because they aren't clickable
+  li.default a:hover .step-indicator {
+    .state-nohover(@wz-color-current);
+  }
+}
 
-        .step-indicator {
-          .state-circle(@big-dot-width, @big-dot-height, @dot-border-width);
-          .state-circle-with-border-and-content(@dot-border-width, @wz-color-default);
-        }
+aw-wizard-navigation-bar.vertical {
+  max-width: 280px;
+  width: 20%;
+  height: 100%;
+  position: sticky;
+  top: 0;
 
-        a:hover .step-indicator {
-          .state-circle-hover(@big-dot-width, @big-dot-height, @dot-border-width);
-          .state-circle-with-border-and-content-hover(@dot-border-width, @wz-color-default);
-        }
+  &.small ul.steps-indicator {
```

Comment:
Sometimes, a user of the library may want to completely override the style of the library, i.e. he doesn't want to use any existing style but define his own one.
In such a case it would be useful to define a kind of "empty" default style.

I'm not sure how this could be done best, and whether this fits in the scope of this PR.
My current approach to such questions was to recommend using a non existing layout name in `[navBarLayout]`, but I'm not sure if this is the best approach.
What do you think about this?

Reply:
My first thought was that providing a default style is a good and easily implemented idea:

```
  ul.steps-indicator {
    .state-default(@color) { }
    .state-hover(@color) { }
    .state-nohover(@color) { }
    .define-style(/* ??? */);
  }
```

...but we will then have to decide which indicator width, height and border-width users are going to need. There will always be users unsatisfied with any assumption we make, so I'd rather not provide any defaults at all to avoid such questions.